app/src/main/java/de/tum/in/tumcampusapp/component/ui/search/adapter/RecentSearchesAdapter.kt

```diff
@@ -54,8 +54,7 @@ class RecentSearchesAdapter(
                             .load(person.getFullImageUrl())
                             .into(profilePicture)
                     } catch (exception: Exception) {
-                        titleText.setText(R.string.not_available_search)
-                        profilePicture.setImageDrawable(null)
+                        onRemove(recentSearch)
```

Comment:
```suggestion
                        Utils.log(exception)
```
When Picasso throws an error in dev and not just silently hide it, or what is the other way this can fail?

Reply:
```
import de.tum.`in`.tumcampusapp.utils.Utils
```
(is required)

Reply:
When I tried to debug that part, the variable is showed as 'Cannot find local variable 'room''. So it throws error.
Is that what you asked about?

Reply:
I again tried to reporoduce this: I cannot reproduce this. Do you know any hints on how to reproduce this?